docker-compose.yml

```diff
@@ -47,6 +47,8 @@ services:
       - AB2D_DB_USER=ab2d
       - AB2D_DB_PASSWORD=ab2d
       - AB2D_EFS_MOUNT=/tmp/ab2d_efs_mount
+      - AB2D_BFD_KEYSTORE_PASSWORD="${AB2D_BFD_KEYSTORE_PASSWORD}"
```

Comment:
You may want to add AB2D_BFD_URL to make in consistent with the variables that are set during the automated deployment of a worker container.

Reply:
👍